design/configuration-crd.md

```diff
@@ -107,7 +107,7 @@ spec:
     cluster:
       dnsLookupFamily: auto
   gateway:
-      controllerName: projectcontour.io/projectcontour/contour
+      controllerName: projectcontour.io/gateway-controller
```

Comment:
i would say we can maybe leave the `design` directory as-is, so these documents capture what the thinking was when they were written, if they need a deliberate update we should do so more explicitly

Reply:
+1

Reply:
done